docs/blog/Webots-2023-a-release.md

```diff
@@ -0,0 +1,98 @@
+# Version R2023a Released
+
+<p id="publish-data">By Yannick Goumaz - XXth November 2022</p>
+
+---
+
+It is that time of the year again!
+Today we are happy to announce the release of Webots R2023a!
+This new version is packed with some new features, improvements and, of course, bug fixes.
+
+Here we are going to present some of the main new features, but for a comprehensive list of changes please refer to the [Change Log](../reference/changelog-r2023.md).
+
+## New Robot
+
+A new robot model has been added to the Webots library. 
+The ROS-based four-wheeled [ROSbot](../guide/rosbot.md) robot from [Husarion](https://husarion.com/) is now included in Webots.
+
+%figure "ROSbot"
+![Added Robots](images/rosbot.thumbnail.png)
+%end
+
+You can discover this new robot in its demo world under `File > Open Sample World > rosbot.wbt`.
+
+---
+
+## New Device Objects
+
+Devices embedded on the [ROSbot](../guide/rosbot.md) have also been added to the Webots library, namely the [Mpu-9250](../guide/imu-sensors.md#mpu-9250) IMU, the [RPLidarA2](../guide/lidar-sensors.md#slamtec-rplidar-a2) lidar and the [Astra](../guide/range-finder-sensors.md#orbbec-astra) RGB-D camera.
+
+%figure "New devices"
+![Added Devices](images/devices_r2023a.thumbnail.png)
+%end
+
+---
+
+## New Assets
+
+Additional models have been added to the Webots library, namely static animals (cow, horse, deer, sheep, dog, fox, cat and rabbit) and a barn.
+
+%figure "New objects"
+![Added Assets](images/assets_r2023a.thumbnail.png)
+%end
+
+---
+
+## New Python API
+
+Webots R2023a comes with a brand new Python API! This API has the advantage of being compatible with all Python distributions (including conda, anaconda, etc.). 
+The API will always be compatible with current and upcoming Python versions.
+
+This change of API does not induce any change to the methods. 
+Python controllers developed before Webots R2023a are fully compatible with this new version.
+
+Moreover, in order to enrich the collection of sample controllers, all sample worlds of the device nodes provide an additional Python controller equivalent to the C controller already included to demonstrate the usage of the API functions.
+
+---
+
+## New IMU Sample World
+
+So far, the [InertialUnit](../reference/inertialunit.md) was considered as an IMU, because it provides the ground truth data of the attitude (roll, pitch, yaw) of the object on which it is grafted. 
```

Comment:
```suggestion
So far, the [InertialUnit](../reference/inertialunit.md) was considered as an IMU, because it provides the ground truth data of the attitude (roll, pitch, yaw) of the object on which it is inserted. 
```

Reply:
grafted is more a botanical term